services/ui-src/src/components/banners/Banner.tsx

```diff
@@ -0,0 +1,55 @@
+// components
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  AlertTitle,
+  Flex,
+} from "@chakra-ui/react";
+
+import { BannerTypes } from "utils/types/types";
+
+// Make alert banner and pass in status and color, text,
+export const Banner = ({
+  status,
+  bgColor,
+  accentColor,
+  title,
+  description,
+}: Props) => {
```

Comment:
We could also have like a `variant` prop or something, and have one each for info, warning, success, error, etc. Could have those be a class or something that would change the colors in our `sx` depending on that variant. Just a thought if we end up needing others of these.

Reply:
Chatted w/ David about banner, he said there is a separate story for it next sprint so this is basically a placeholder for now - we need to address banner text with mobile view as well.